dataset/web-scrapers/commandlinefu.py

```diff
@@ -0,0 +1,31 @@
+import requests
+from bs4 import BeautifulSoup
+
+TOTAL_COMMANDS = 14042
+TOP_COMMANDS = 500
```

Comment:
`TOTAL_COMMANDS` is obtained by automatically by running the parser until nothing is returned.
`TOP_COMMANDS` is an arbitrary number to obtain the first 500 most popular posts because we don't need all the posts hosted on the website.

Reply:
Could you please add those as comments